go.mod

```
@@ -0,0 +1,5 @@
+module github.com/elastic/go-perf
+
+go 1.20
```

Allow a bit more backwards compatibility.
```suggestion
go 1.18
```